### ISSUE TITLE
Mardown anchors support

### ISSUE THREAD

**Issue by gospodima** [KyleKing/mdformat-mkdocs#25]
Hello,

thank you for maintaining this library!

I am using [markdown anchors](https://mkdocstrings.github.io/autorefs/#markdown-anchors) from [autorefs](https://mkdocstrings.github.io/autorefs/) plugin.

mdformat changes `[](){#some-anchor-name}` to `[](<>){#some-anchor-name}`. Is it possible to disable such behaviour?

.pre-commit-config.yaml:

```yaml
repos:
- repo: https://github.com/executablebooks/mdformat
  rev: 0.7.17
  hooks:
    - id: mdformat
      additional_dependencies:
        - mdformat-mkdocs[recommended]>=2.0.0
      args: [--wrap=88, --number]
```

**Comment by KyleKing** [comment]
`mdformat-mkdocs` doesn't modify links directly, so my guess is that this is the result of an indirect dependency (likely [`linkify`](https://github.com/executablebooks/markdown-it-py/blob/1064835b30325e857252f5b67b34e622261723ca/markdown_it/rules_core/linkify.py) through `mdformat-gfm`)

I will have some time later today to investigate and I'll follow once I can figure out the root cause

**Comment by KyleKing** [comment]
I've been trying to find a better way to fix rendering of these links, but I have a workable solution and released the change in v2.0.11. Let me know if you run into any issues!

**Comment by pelson** [comment]
Looks like it has done the trick. Thanks!
